use_cases/eluc/predictors/neural_network/neural_net_predictor.py

```diff
@@ -79,57 +78,48 @@ class NeuralNetPredictor(Predictor):
     in order to take advantage of the linear relationship in the data.
     Data is automatically standardized and the scaler is saved with the model.
     """
-    def __init__(self, features=None, label=None, hidden_sizes=[4096], linear_skip=True,
-                 dropout=0, device="mps", epochs=3, batch_size=2048, optim_params={},
-                 train_pct=1, step_lr_params={"step_size": 1, "gamma": 0.1}):
-
-        self.features=None
-        self.label=None
-
-        self.set_params(features, label, hidden_sizes, linear_skip,
-                        dropout, device, epochs, batch_size, optim_params,
-                        train_pct, step_lr_params)
+    def __init__(self, model_config: dict):
```

Comment:
A docstring explaining what the config dict should contain would be useful here.